### ISSUE TITLE
Popover: Inline rendering issue

### ISSUE THREAD

**Issue by gilbugauisan** [infor-design/enterprise-ng#1609]
**Describe the bug**
Popover (tooltip) not rendering inline html

**To Reproduce**
1. Go to http://localhost:4200/ids-enterprise-ng-demo/tooltip
2. Click on 'Popover Inline' button
3. See popover

**Actual behavior**
Inline HTML tag not properly rendered

**Expected behavior**
It should display the inline HTML properly

**Version**
- ids-enterprise-ng: 17.2.0-beta.0
- ids-enterprise: 4.90.0

**Screenshots**
![image](https://github.com/infor-design/enterprise-ng/assets/74726741/4a3c14ed-f869-445c-b309-7c2f1536e4e0)

![popover-inline](https://github.com/infor-design/enterprise-ng/assets/74726741/7fcd8d09-96cc-4b33-b414-bde02c069d64)

**Platform**
 - Device (if applicable): Laptop
 - OS Version: Windows 11
 - Browser Name: Edge
 - Browser Version: Version 120.0.2210.77 (Official build) (64-bit)

**Comment by tmcconechy** [comment]
Yeah wierd. Some sort of issue there. For what its worth you could try and look at the main examples and see if you can get it to work https://github.com/infor-design/enterprise/tree/main/app/views/components/popover

Added to (rather large) backlog

**Comment by tjamesallen15** [comment]
I'll try to check this on my Windows 10 PC

**Comment by gilbugauisan** [comment]
> Yeah wierd. Some sort of issue there. For what its worth you could try and look at the main examples and see if you can get it to work https://github.com/infor-design/enterprise/tree/main/app/views/components/popover
> 
> Added to (rather large) backlog

Hi @tmcconechy I checked this [example](https://main-enterprise.demo.design.infor.com/components/popover/test-content-string.html) and this [example](https://main-enterprise.demo.design.infor.com/components/popover/example-extra-css-class.html) and both are not working.

I just found another [bug](https://main-enterprise.demo.design.infor.com/components/popover/example-extra-css-class.html) related to it. When we pass 'extraClass' settings, it doesn't render the popover at all; this includes the header title and content.

Let me know if I've missed something, or anything that is needed on our side so your team can proceed.

We have a critical ticket dependent on it (field history auditing), and the customer is asking us to prioritize this issue for this release (April Release).

Thank you very much.
